### PR TITLE
[READY TO MERGE] Map Updates, Huragok Changes and More

### PR DESCRIPTION
**Pull request by Aroliacue**
:cl: Aroliacue
maptweak: ONI base expanded with a new surgery room, AI core, hydroponics lab and a redesigned cargo bay.
rscadd: Gives Huragoks RCDs.
/:cl:

### CODE REVIEW

**Comment by Aroliacue** [comment]
Heading to bed, will finish this tomorrow

File: code/modules/halo/unsc/supply/onispecial.dm [hunk 1]
```diff
@@ -11,6 +11,24 @@
...
+/decl/hierarchy/supply_pack/oni_special/orion
+	name = "Orion Surgical Supplies"
+	containertype = /obj/structure/closet/crate/freezer //Needs a freezer so organs don't decay
+	contains = list(
+	/obj/item/organ/internal/eyes/occipital_reversal/theta = 1,
+	/obj/item/organ/internal/heart/spartan/theta = 1,
+	/obj/item/organ/internal/liver/spartan/theta = 1,
+	/obj/item/organ/internal/lungs/theta = 1
+	)
+	cost = 3500
```
Comment (ghost) [inline code/modules/halo/unsc/supply/onispecial.dm:23]: Too little for an entire orion conversion kit. And considering the fact that money can be made with ease...
~~Unless covenant gets something special as well, but this ain' happening~~

Comment (Aroliacue) [reply code/modules/halo/unsc/supply/onispecial.dm:23]: XO's working on an item that will allow us to convert people to Orions without admin intervention. ONI already has access to two sets of Orion organs, and a successful Orion surgery takes around 10-15 minutes with the requirements of an ONI researcher (to get the organs) and the know-how on how to do the surgery without killing them. The time and skill investment that you get in return is 20 extra health, 10% pain resist and 0.05 extra movespeed.

The price will likely increase when XO finishes their conversion item, but the Covvies are definitely not getting a equivalent as they already have an extremely varied roster of powerful species and roles.

**Comment by Aroliacue** [comment]
Ready to merge.

**Review by BDpuffy420 — APPROVED** [review]

**Pull request merged**